### PR TITLE
Fix playlist deletion

### LINKED ISSUE #2770: [Bug] Database error trying to delete a playlist
**Describe the bug**
Unable to delete added youtube playlists

**Steps to Reproduce**

1. Go to a youtube playlist
2. Click on '+'
3. Click on trash can to delete
4. Click on 'Yes'
5. See error

**Logs**

Title: `cannot insert multiple commands into a prepared statement (PQ::PQError)`
Date: `2022-01-06T00:06:36Z`
Route: `/delete_playlist?list=PLu9l40IymKw_tT4K9E0lQ2XUxJtTTv4K9&referer=%2Fplaylist%3Flist%3DPLu9l40IymKw_tT4K9E0lQ2XUxJtTTv4K9`
Version: `2022.01.05-3bb7fbb2 @ master`
<details>
<summary>Backtrace</summary>
<p>
   
```
cannot insert multiple commands into a prepared statement (PQ::PQError)
  from lib/pg/src/pq/connection.cr:203:7 in 'handle_error'
  from lib/pg/src/pq/connection.cr:186:7 in 'handle_async_frames'
  from lib/pg/src/pq/connection.cr:162:7 in 'read'
  from lib/pg/src/pq/connection.cr:157:7 in 'read'
  from lib/pg/src/pq/connection.cr:414:31 in 'expect_frame'
  from lib/pg/src/pq/connection.cr:413:5 in 'expect_frame'
  from lib/pg/src/pg/statement.cr:18:5 in 'perform_query'
  from lib/pg/src/pg/statement.cr:35:14 in 'perform_exec'
  from lib/db/src/db/statement.cr:85:9 in 'perform_exec_and_release'
  from lib/db/src/db/statement.cr:70:7 in 'exec:args'
  from lib/db/src/db/pool_statement.cr:19:30 in 'exec:args'
  from lib/db/src/db/query_methods.cr:275:7 in 'exec'
  from src/invidious/database/playlists.cr:33:5 in 'delete'
  from src/invidious/routes/playlists.cr:129:5 in 'delete'
  from src/invidious.cr:318:1 in '->'
  from /usr/share/crystal/src/primitives.cr:266:3 in '->'
  from /usr/share/crystal/src/primitives.cr:266:3 in 'process_request'
  from lib/kemal/src/kemal/route_handler.cr:17:7 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/websocket_handler.cr:13:14 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/filter_handler.cr:21:7 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/handlers.cr:212:5 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/handlers.cr:94:12 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/handlers.cr:145:12 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/handlers.cr:70:5 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/static_file_handler.cr:112:11 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/exception_handler.cr:8:7 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/logger.cr:17:35 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/init_handler.cr:12:7 in 'call'
  from /usr/share/crystal/src/http/server/request_processor.cr:51:11 in 'process'
  from /usr/share/crystal/src/http/server.cr:515:5 in 'handle_client'
  from /usr/share/crystal/src/http/server.cr:468:13 in '->'
  from /usr/share/crystal/src/primitives.cr:266:3 in 'run'
  from /usr/share/crystal/src/fiber.cr:92:34 in '->'
  from ???
```
</p>
</details>

**Additional context**
Running on current master https://github.com/iv-org/invidious/commit/3bb7fbb2f119790ee6675076b31cd990f75f64bb

### PR DESCRIPTION
**Pull request by matthewmcgarvey**
Fixes #2770

Can't execute two statements through `PG_DB.exec`

### CODE REVIEW

**Comment by matthewmcgarvey** [comment]
Looked through other database modules and don't see any others that tried this

File: src/invidious/database/playlists.cr [hunk 1]
```diff
@@ -207,6 +206,15 @@ module Invidious::Database::PlaylistVideos
     PG_DB.exec(request, index)
   end
 
+  def delete_by_playlist(playlist_id : String)
+    request = <<-SQL
+      DELETE FROM playlist_videos *
+      WHERE plid = $1;
```
Comment (SamantazFox) [inline src/invidious/database/playlists.cr:212]:
```suggestion
      WHERE plid = $1
```

File: src/invidious/database/playlists.cr [hunk 2]
```diff
@@ -207,6 +206,15 @@ module Invidious::Database::PlaylistVideos
     PG_DB.exec(request, index)
   end
 
+  def delete_by_playlist(playlist_id : String)
```
Comment (SamantazFox) [inline src/invidious/database/playlists.cr:209]:
```suggestion
  def delete_by_playlist(plid : String)
```

Comment (SamantazFox) [reply src/invidious/database/playlists.cr:209]: In order to be consistent with the other functions :)

**Review by SamantazFox — APPROVED** [review]

**Pull request merged**